t/discovery/eureka.t

```diff
@@ -26,7 +26,8 @@ apisix:
   node_listen: 1984
   config_center: yaml
   enable_admin: false
-  discovery: eureka
+  discovery:
+    - eureka
 
```

Comment:
We can also move all eureka configuration to here, it would be better.

Reply:
done